packages/orchestrator-ui-components/src/components/WfoForms/formFields/ListDelField.tsx

```diff
@@ -49,20 +49,19 @@ function ListDel({
         { absoluteName: true },
     )[0];
 
-    const limitNotReached =
-        parent &&
+    // Make a row deletable only if its:
+    // - doing so doesn't go below the minCount for the field
+    // - it's not the first one. You should be able to remove an empty row
+    const isDeletable: boolean =
         !disabled &&
-        parent.minCount &&
-        parent.value &&
-        !(parent.minCount >= parent.value.length);
+        !readOnly &&
+        (!parent.minCount ||
+            (parent.value && parent.value.length >= parent.minCount))
+            ? true
+            : false;
```

Comment:
Removing these 2 lines should end up in the same result

Reply:
This ensures a possible result is not undefined